tests/complete/main.tf

```diff
@@ -210,6 +172,14 @@ module "eks_blueprints_addons" {
   ## An S3 Bucket ARN is required. This can be declared with or without a Prefix.
   velero = {
     s3_backup_location = "${module.velero_backup_s3_bucket.s3_bucket_arn}/backups"
+    values = [
+      # https://github.com/vmware-tanzu/helm-charts/issues/550#issuecomment-1959933230
```

Comment:
I don't want to create another version mapping like cluster-autoscaler, so just applying to fix in the test case for now while waiting for the upstream project to find a viable solution